server/src/s3.rs

```diff
@@ -171,6 +173,8 @@ impl S3 {
             .region(options.region)
             .endpoint_resolver(options.endpoint)
             .credentials_provider(options.creds)
+            .retry_config(RetryConfig::standard().with_max_attempts(1))
```

Comment:
@nitisht  Any other suggestion on max_attempts value

Reply:
Let's change it to 5. Higher retries is generally safer.